service/service_account.go

```diff
@@ -42,30 +43,30 @@ func (s AccountService) AccountBalance(
 		return nil, wrapError(errInvalidInput, "account identifier is not provided")
 	}
 
-	header, err := blockHeaderFromInput(ctx, s.client, req.BlockIdentifier)
-	if err != nil {
-		return nil, wrapError(errInternalError, err)
+	header, terr := blockHeaderFromInput(ctx, s.client, req.BlockIdentifier)
```

Comment:
Why make these error vars less descriptive?

Reply:
It's a stylistic choice, we don't do this anywhere else